test/OpenTelemetry.Contrib.Shared.Tests/SqlProcessorAdditionalTestCases.json

```diff
@@ -883,5 +883,31 @@
       ],
       "db.query.summary": "SELECT TableA TableB"
     }
+  },
+  {
+    "name": "bracketed_identifier",
+    "input": {
+      "db.system.name": "other_sql",
+      "query": "SELECT [a], [b] FROM [TableA], [TableB]"
+    },
+    "expected": {
+      "db.query.text": [
+        "SELECT [a], [b] FROM [TableA], [TableB]"
+      ],
+      "db.query.summary": "SELECT TableA TableB"
+    }
+  },
+  {
+    "name": "bracketed_identifier_with_single_quote",
+    "input": {
+      "db.system.name": "other_sql",
+      "query": "SELECT [a], [b] FROM [Table'A], [TableB]"
+    },
+    "expected": {
+      "db.query.text": [
+        "SELECT [a], [b] FROM [Table"
+      ],
+      "db.query.summary": "SELECT Table"
+    }
```

Comment:
Rather than add a test case that passes when it shouldn't, it might be better to add metadata we can use in the tests to skip it because we know it's broken.

For example, a `"skip": "https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/3574"` property?

Then the test can skip itself if the property is present with a non-empty value.